client/src/modules/list/index.jsx

```diff
@@ -315,6 +398,42 @@ class List extends Component {
             )}
           />
         )}
+        {idDialogForRemovedListVisible && (
+          <Dialog
+            cancelLabel="common.button.dashboard"
+            confirmLabel="common.button.restore"
+            hasPermissions={isOwner}
+            onCancel={this.handleRedirect}
+            onConfirm={isArchived && !isDeleted ? this.handleRestoreList : null}
+            pending={pendingForListRestoring}
+            title={formatMessage(
+              {
+                id: 'list.actions.not-available'
+              },
+              { name }
+            )}
+          >
+            <p style={{ whiteSpace: 'pre-line' }}>
+              <FormattedMessage
+                id={externalAction.messageId}
+                values={{
+                  name,
+                  cohortName,
+                  performer: externalAction.performer
+                }}
+              />
+              {!isOwner && (
+                <Fragment>
+                  {' '}
```

Comment:
is this necessary?

Reply:
We join two spans iN the dialog without this There is no space between them. I tried to use styles:
https://github.com/sandstreamdev/eoc/pull/363#discussion_r329534033